### PR TITLE
Sync changes from UPP for GFSV16

### LINKED ISSUE #421: Sync the changes from UPP for GFSV16
Update the branch develop with the changes from UPP version for GFSV16:

- Remove mange_externals for checking gtg code
- Check out gtg code as submodule
- Update GFS post workflow scripts with changes from GFSV16

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
The main changes are:

- Remove mange_externals for checking gtg code

- Check out gtg code as submodule

- Update GFS post workflow scripts with changes from GFSV16

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
The UPP regression tests were conducted on WCOSS-Dell and WCOSS-Cray. All tests were passed.

File: scripts/exgdas_atmos_nceppost.sh [hunk 1]
```diff
@@ -16,6 +16,8 @@ echo " Feb 18 - Meng - Removed legacy setting for generating grib1 data"
 echo "                 and reading sigio model outputs."
 echo " Aug 20 - Meng - Remove .ecf extentsion per EE2 review."
 echo " Sep 20 - Meng - Update clean up files per EE2 review."
+echo " Oct 21 - Meng - Remove jlogfile for wcoss2 transition."
+echo " Nov 21 - Meng - Update POSTGRB2TBL default setting for wcoss2 transition."
```
Comment (HuiyaChuang-NOAA) [inline scripts/exgdas_atmos_nceppost.sh]: @WenMeng-NOAA don't see this change in the script part, just want to double check with you

Comment (WenMeng-NOAA) [reply scripts/exgdas_atmos_nceppost.sh]: @HuiyaChuang-NOAA Thanks for catching this. Actually the changes were already applied but the change logs were not updated. I tweaked them now.

**Review by HuiyaChuang-NOAA — APPROVED** [review]
Looks good.  Thank you

**Comment by WenMeng-NOAA** [comment]
The UPP regression tests were conducted on WCOSS1-Dell, WCOSS1-Cray and Here. No changed results were found.

**Pull request merged**